cmd/influx/bucket.go

```diff
@@ -179,6 +183,13 @@ func (b *cmdBucketBuilder) cmdList() *cobra.Command {
 			EnvVar: "BUCKET_NAME",
 			Desc:   "The bucket name",
 		},
+		{
+			DestP:  &b.limit,
+			Flag:   "limit",
+			Short:  'l',
+			EnvVar: "LIMIT",
+			Desc:   "The max number of results to return (must be < 100)",
```

Comment:
We should probably add this to all the find / list commands

Reply:
What happens if a user passes > 100? Does the server just return the first 100 things, or does something go boom?

Reply:
they get a message: `Error: Failed to retrieve buckets: limit must be between 1 and 100.`